src/LibraryManager/Providers/Unpkg/NpmPackageSearch.cs

```diff
@@ -96,52 +97,98 @@ private static async Task<IEnumerable<string>> GetPackageNamesWithScopeAsync(str
 
         private static async Task<IEnumerable<string>> GetPackageNamesFromSimpleQueryAsync(string searchTerm, CancellationToken cancellationToken)
         {
-            List<string> packageNames = new List<string>();
+            string searchUrlFormat = GetCustomNpmRegistryUrl() ?? NpmPackageSearchUrl;
```

Comment:
LibMan queries NPM to get package info that comes from CDNs (unpkg, jsdelivr) - they won't be able to serve up packages from custom feeds.  Since the GetCustomNpmRegistryUrl() method is basically a noop, should we just remove it?  It doesn't feel like it makes sense in this code base.

Reply:
Yeah, I agree, I removed that.